conf/solution/config.yaml.sample

```diff
@@ -0,0 +1,162 @@
+cortx:
+  external:
+    kafka:
+      endpoints:
+        - tcp://kafka.default.svc.cluster.local:9092
+      admin: admin
+      secret: kafka_admin_secret
+    consul:
+      endpoints:
+        - tcp://consul-server.default.svc.cluster.local:8301
+        - http://consul-server.default.svc.cluster.local:8500
+      admin: admin
+      secret: consul_admin_secret
+  common:
+    release:
+      name: CORTX
+      version: 2.0.0-1111
+    setup_size: small
+    service:
+      admin: admin
+      secret: common_admin_secret
+    storage:
+      log: /var/log/cortx/
+      local: /etc/cortx
+      config: /etc/cortx
+    security:
+      ssl_certificate: /etc/cortx/solution/ssl/s3.seagate.com.pem
+      domain_certificate: /etc/cortx/solution/ssl/stx.pem
+      device_certificate: /etc/cortx/solution/ssl/stx.pem
+  utils:
+    message_bus_backend: kafka
+  rgw:
+    s3:
+      endpoints:
+      - http://cortx-io-svc:80
```

Comment:
same comment applies, whats mentioned in gconf

Reply:
changed the endpoints by adding product name